libsolidity/codegen/ir/IRGenerator.h

```diff
@@ -54,12 +54,14 @@ class IRGenerator
 	/// (or just pretty-printed, depending on the optimizer settings).
 	std::pair<std::string, std::string> run(
```

Comment:
At some point we should document the inputs (contract, metadata, otherYulSources), but doesn't need to be this PR. Not sure what otherYulSources is though?

Reply:
other yul sources is the mapping of sources for contracts this contract creates.

Reply:
At some point we should create some state object out of the arguments.